modules/openapi-generator/src/main/resources/aspnetcorelib/2.1/tupleReturn.mustache

```diff
@@ -0,0 +1,8 @@
+var (statusCode, result) = await _impl.{{operationId}}({{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}});
```

Comment:
Why the use of tuples and an extractor here? Can this be rolled into a simple object, to ease evolution of the generated code?

Reply:
The main reason was that it was simple and native to .Net. 

While we can roll it into an object, Tuples do the same thing with less code. However, can also see value in an object being returned.